extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -27,7 +32,7 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		for (id in registeredCallbacks) {
 			if (registeredCallbacks.hasOwnProperty(id)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the nested one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)